### DISCUSSION TITLE
No update to 2.1.2

### DISCUSSION THREAD

**Discussion by craft-and-code** [emmercm/igir#847] [General]
Usually, when I call `npx igir`, and update is available, I can update to the last one.
Not now. I tried `npx igir@latest`, but I am still in 2.1.1

It's like your last update is not recognised.

**Comment by emmercm** [comment]
You're absolutely right, it looks like the publish workflow is failing and I'm not entirely sure why. I'll need to debug this in the next day or two.

**Reply by emmercm** [reply]
Looks like I fixed it here: https://github.com/emmercm/igir/pull/848

v2.1.2 just published to npm successfully.

**Reply by craft-and-code** [reply]
It's works, thanks :)
